code/__DEFINES/robots.dm

```diff
@@ -7,6 +7,49 @@
 #define LAW_ION "ion"
 #define LAW_HACKED "hacked"
 
+//AI notification defines
+///Alert when a new Cyborg is created.
+#define NEW_BORG 1
```

Comment:
Can we please prefix all of these with something, some of them like `RENAME` and `DISCONNECT` are super generic